R/uvhydrograph-data.R

```diff
@@ -20,19 +21,23 @@ parseUVData <- function(data, plotName, month) {
     meas_Q <- subsetByMonth(getFieldVisitMeasurementsQPoints(data), month)  
     
     primary_lbl <- getUvLabel(data, "primarySeries")
+    dvmin_lbl <- getUvLabel(data, "derivedSeriesMin")
```

Comment:
I then tried to wrap something around that function so it wouldn't enter it, but couldn't get that figured out, so went back to just modifying how I have in the latest.